applications/portfolioDrafts/deletePortfolioDraft.ts

```diff
@@ -0,0 +1,46 @@
+import { DeleteCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { parse } from "uuid";
+import { dynamodbClient as client } from "./utils/dynamodb";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+// const PRIMARY_KEY = process.env.PRIMARY_KEY || "";
+
+/**
+ * Deletes a Portfolio Draft
+ *
+ * @param event - The DELETE request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (portfolioDraftId == null) {
+    return { statusCode: 400, body: "Bad Request. A Portfolio Draft ID must be specified in URL path." };
```

Comment:
We should talk about defining an error result type.

```
{
  "errorCode": "InvalidRequest",
  "errorMessage": "A valid portfolio draft ID must be specified"
}
```

And maybe some helper functions? Eventually, it may be useful for the front end to parse info about errors so we'll want an expandable format for that.